tools/niminst/install.tmpl

```diff
@@ -3,7 +3,16 @@
 #  result = "#! /bin/sh\n# Generated by niminst\n"
 #  var proj = c.name.toLower
 
-set -e
+BASE_DIR=$(pwd)
+NIM_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function go_back {
```

Comment:
I don't think 'function' is in the Posix standard. Note that this script is Posix, not Bash.